README.md

```diff
@@ -141,21 +141,22 @@ languages and platforms.
 
 ## Supported Languages
 
-* [Java](java/app-encryption)
+* [Java](https://github.com/godaddy/asherah-java)
 * [.NET](csharp/AppEncryption)
 * [Go](go/appencryption)
 * [Service Layer (gRPC)](/server)
+* [Node.JS](https://github.com/godaddy/asherah-node)
+* [Ruby](https://github.com/godaddy/asherah-ruby)
+* [Python](https://github.com/godaddy/asherah-python)
```

Comment:
We need something here to differentiate between the actual SDK implementations found in this repo, i.e., Java, .NET, Go, and Service Layer (aka sidecar), and the Cobhan-based alternatives which are derived from the Go implementation. A new section perhaps?

```suggestion

### Derived Projects

[Asherah-Cobhan](https://github.com/godaddy/asherah-cobhan) implementations built atop the [Asherah SDK for Go](go/appencryption).

* [Node.JS](https://github.com/godaddy/asherah-node)
* [Ruby](https://github.com/godaddy/asherah-ruby)
* [Python](https://github.com/godaddy/asherah-python)
```